src/db/models/Contact.ts

```diff
@@ -0,0 +1,53 @@
+import { Model, DataTypes } from "sequelize";
+
+import sequelize from "./index";
+
+export default class Contact extends Model {
+    public id!: number;
+
+    public userId!: string;
+
+    public type!: string;
```

Comment:
is it not supposed to be an index?

Reply:
@ComeTiss Not sure...  I don't think so, Sequelize enums ensures that the string provided matches a set of values. See [link](https://sequelize.readthedocs.io/en/2.0/docs/models-definition/)
I agree with you, I thought that the point was to provide a int as it requires less memory than a string, but I might have got it wrong. Any thought ?